pkg/pillar/cmd/zedagent/parseEdgeview.go

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) 2021-2022 Zededa, Inc.
```

Comment:
I think we avoid camelcase in names of the source files.

Reply:
sure, will change to all lower case.